Traceback (most recent call last):
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/fedKMeans/HPC_scripts/../main.py", line 37, in <module>
    results = run(config)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/fedKMeans/fed_kmeans.py", line 160, in run
    clients, local_clusters, cluster_sizes = load_clients(config)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/fedKMeans/fed_kmeans.py", line 127, in load_clients
    data, labels = load_data(i, config['dset'], beta=config['beta'], ppc = config['ppc'], noise=config['noise'])
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/fedKMeans/common.py", line 32, in load_data
    data = np.genfromtxt(datafile, delimiter=',')
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/flower/lib/python3.10/site-packages/numpy/lib/npyio.py", line 1980, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/flower/lib/python3.10/site-packages/numpy/lib/_datasource.py", line 193, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/flower/lib/python3.10/site-packages/numpy/lib/_datasource.py", line 533, in open
    raise FileNotFoundError(f"{path} not found.")
FileNotFoundError: /tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/fedKMeans/data/beta10.0/cluster_beta10.0_ppc50_noise1_client0.csv not found.
srun: error: gpu09: task 198: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=10670372.0
slurmstepd: error: *** STEP 10670372.0 ON gpu07 CANCELLED AT 2024-07-01T18:03:54 ***
slurmstepd: error: common_file_write_content: unable to open '/sys/fs/cgroup/freezer/slurm/uid_618619/job_10670372/step_0/freezer.state' for writing: Permission denied
srun: error: gpu07: tasks 0-31: Terminated
srun: error: gpu09: tasks 124-197,199: Terminated
srun: error: gpu08: tasks 32-123: Terminated
srun: Force Terminated StepId=10670372.0
